pkg/strategy/supertrend/strategy.go

```diff
@@ -478,14 +494,59 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 
 			s.AccumulatedProfitReport.RecordProfit(profit.Profit)
 		})
-		s.orderExecutor.TradeCollector().OnTrade(func(trade types.Trade, profit fixedpoint.Value, netProfit fixedpoint.Value) {
-			s.AccumulatedProfitReport.RecordTrade(trade.Fee)
-		})
+		// s.orderExecutor.TradeCollector().OnTrade(func(trade types.Trade, profit fixedpoint.Value, netProfit fixedpoint.Value) {
+		// 	s.AccumulatedProfitReport.RecordTrade(trade.Fee)
+		// })
```

Comment:
wonder the reason to remove it? maybe just remove the block if it's not used anymore?

Reply:
Removed (OnTrade is still in use, but not only for backtesting)